packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -198,14 +198,16 @@ class TextPainter {
 
   bool get _debugNeedsLayout => _paragraph == null;
```

Comment:
We should then probably remove this flag and just use the _needsLayout flag everywhere?

Reply:
I think `_needsLayout` is not exactly equal to ` _paragraph == null` , in some cases `_needsLayout` is true but `_paragraph` is not null , using it everywhere will still result in #84650, @goderbauer